src/renderers/dom/shared/DOMPropertyOperations.js

```diff
@@ -130,7 +130,7 @@ var DOMPropertyOperations = {
    * @param {string} name
    * @param {*} value
    */
-  setValueForProperty: function(node, name, value) {
+  setValueForProperty: function(node, name, value, isCustomElement) {
```

Comment:
Not sure if there is a better way to do this. I want custom properties on custom elements to be set, but [the next few lines of code](https://github.com/facebook/react/pull/8755/files#diff-f04a397aa63ac69c28dab46e9863bb90R134) prevent that from happening because DOMProperties.properties cannot possible have a propertyInfo for a custom property name.

The reason why I'm calling this function at all is because I want the [instrumentation code in this function](https://github.com/facebook/react/pull/8755/files#diff-f04a397aa63ac69c28dab46e9863bb90R171) to run. Otherwise I would consider just doing `node[propName] = propValue` inside of ReactDOMComponent itself.